java-webhooks-example/README.md

```diff
@@ -4,39 +4,48 @@
 
 This is a simple web application that uses the WorkOS Java SDK to validate and display webhooks sent from WorkOS in real time.
 
-## Running 
+## Running
 
 ### 1. Set Up Local WorkOS API Key
-Log into the [WorkOS Dashboard](https://dashboard.workos.com/api-keys) and set the `WORKOS_API_KEY` environment variable to your WorkOS API Key
 
-```sh
-export WORKOS_API_KEY={{YOUR_API_KEY}}
+Log into the [WorkOS Dashboard](https://dashboard.workos.com/api-keys) and locate your WorkOS API Key.
+
+Then create a `.env` file at the root of the project and populate the following environment variable.
+
+```
+WORKOS_API_KEY=your_api_key_here
 ```
 
-### 2. Start an `ngrok` session 
+### 2. Start an `ngrok` session
 
 [Ngrok](https://ngrok.com/) is a simple application that allows you to map a local endpoint to a public endpoint.
 
 The application will run on http://localhost:7005. Ngrok will create a tunnel to the application so we can receive webhooks from WorkOS.
+
 ```sh
 ./ngrok http 7005
 ```
 
 ### 3. Set Up a WorkOS Endpoint
+
 Log into the [WorkOS Dashboard](https://dashboard.workos.com/webhooks) and add a Webhook endpoint with the public ngrok URL with `/webhooks` appended.
 
 The local application is listening for webhook requests at http://localhost:7005/webhooks
 
 ### 4. Set Up Webhooks Secret
-In order for the SDK to validate that WorkOS webhooks, set the Webhook secret from the dashboard into the `WORKOS_WEBHOOK_SECRET` environment variable.
+
+In order for the SDK to validate that WorkOS webhooks, locate the Webhook secret from the dashboard.
+
+Then populate the following environment variable in your `.env` file at the root of the project.
+
 ```sh
-export WORKOS_WEBHOOK_SECRET={{YOUR_WEBHOOK_SECRET}}
+WORKOS_WEBHOOK_SECRET=your_webhook_secret
```

Comment:
I think a `WORKOS_API_KEY` is required here too (oops). Because we require it in the SDK, even though we may not use it for webhooks.